trinity/cli_parser.py

```diff
@@ -23,28 +25,107 @@ class ValidateAndStoreEnodes(argparse.Action):
     def __call__(self,
                  parser: argparse.ArgumentParser,
                  namespace: argparse.Namespace,
-                 values: Any,
+                 value: Any,
                  option_string: str=None) -> None:
-        if values is None:
+        if value is None:
             return
 
-        enode = Node.from_uri(values)
+        enode = Node.from_uri(value)
 
         if getattr(namespace, self.dest) is None:
             setattr(namespace, self.dest, [])
         enode_list = getattr(namespace, self.dest)
         enode_list.append(enode)
 
 
+LOG_LEVEL_CHOICES = {
+    # numeric versions,
+    '5': TRACE_LEVEL_NUM,
+    '10': logging.DEBUG,
+    '20': logging.INFO,
+    '30': logging.WARNING,
+    '40': logging.ERROR,
+    '50': logging.CRITICAL,
+    # lowercase versio,
+    'trace': TRACE_LEVEL_NUM,
+    'debug': logging.CRITICAL,
+    'info': logging.INFO,
+    'warn': logging.WARNING,
+    'warning': logging.WARNING,
+    'error': logging.ERROR,
+    'critical': logging.CRITICAL,
+    # capcase versio,
```

Comment:
Instead of maintaining upper and lower case versions I think it would be better to simply decide on one casing and then do `value.lower()` or `value.upper()` before the lookup.